src/go/vtunnel/pkg/vmsock/constants.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright © 2022 SUSE LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vmsock
+
+const (
+	HostListenPort    = uint32(6969)
+	PeerHandshakePort = uint32(8989)
+	SeedPhrase        = "github.com/rancher-sandbox/rancher-desktop-vtunnel"
```

Comment:
These things shouldn't be constant, right? Otherwise if we have two different processes trying to establish vsock tunnels they'll trample over each other.

Reply:
Oh, of course. I totally forgot we are going to re-use this :b